### PR TITLE
Docs for Int

### PR DESCRIPTION
**Pull request by aspeddro**
Adapted from https://github.com/rescript-lang/rescript-compiler/blob/master/jscomp/others/js_int.ml

### CODE REVIEW

File: src/Core__Int.resi [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+`v`. `~radix` specifies the radix base to use for the formatted number.
+
+## Examples
+
+```rescript
+Int.fromString("0") == Some(0)
+Int.fromString("NaN") == None
+```
+*/
+let fromString: (~radix: int=?, 'a) => option<int>
```
Comment (aspeddro) [inline src/Core__Int.resi:262]: This function signature should not be `(string, ~radix: int=?) => option<int>`?

File: src/Core__Int.resi [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+```rescript
+Int.toFloat(100) == 100.
+Int.toFloat(2) == 2.0
+```
+*/
+external toFloat: int => float = "%identity"
+
+/**
+`fromFloat(n)` return an `int` representing the given value.
```
Comment (zth) [inline src/Core__Int.resi:241]: Maybe add a short sentence on how it rounds the underlying float? And add an example or two for that. `Int.fromFloat(1.5)` and `Int.fromFloat(1.3)` for example.

File: src/Core__Int.resi (hunk 1 above)
Comment (zth) [reply src/Core__Int.resi:263]: Yeah that seems more logical. Although we should preserve the current order of the arguments.

File: src/Core__Int.resi [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+/**
+`fromString(~radix?, v)` return an `option<int>` representing the given value
+`v`. `~radix` specifies the radix base to use for the formatted number.
+
+## Examples
+
+```rescript
+Int.fromString("0") == Some(0)
+Int.fromString("NaN") == None
```
Comment (zth) [inline src/Core__Int.resi:260]: Maybe add an example using another radix as well?

**Review by zth — COMMENTED** [review]
Looking great! 😄 Just a few minor comments.

**Review by zth — APPROVED** [review]

**Pull request merged**